pkg/metric/metrics_service.go

```diff
@@ -56,16 +56,18 @@ func (metricService *MetricService) UpdateCompletedTaskCount(ctx context.Context
 }
 
 func (metricService *MetricService) UpdateTotalTaskResultsCount(ctx context.Context) error {
-	cache := cache.GetCacheInstance()
+	cacheInstance := cache.GetCacheInstance()
```

Comment:
can we keep it consistent, in previous PRs `cache := cache.GetCacheInstance()` was used but here `cacheInstance := cache.GetCacheInstance()` is used